core/src/main/java/io/seata/core/exception/TransactionExceptionCode.java

```diff
@@ -118,7 +118,17 @@
     /**
      * Failed to store exception code
      */
-    FailedStore
+    FailedStore,
+
+    /**
+     * Participant rollback done
+     */
+    ParticipantRollbackDone,
+
+    /**
+     * Participant rollback failure
+     */
+    ParticipantRollbackFailure
```

Comment:
please move them to TransactionalExecutor.Code, then implement them in GlobalTransactionalInterceptor.handleGlobalTransaction.

Reply:
> please move them to TransactionalExecutor.Code, then implement them in GlobalTransactionalInterceptor.handleGlobalTransaction.

ok